pkg/schemas/secret_gen.go

```diff
@@ -0,0 +1,69 @@
+// Code generated by sdk-to-schema generator; DO NOT EDIT.
+
+package schemas
+
+import (
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/sdk"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+// ShowSecretSchema represents output of SHOW query for the single Secret.
+var ShowSecretSchema = map[string]*schema.Schema{
```

Comment:
please check user_ext.go file in this package, it shows how to add sensitivity to the generated schemas (I am not sure if any of these fields - or in the describe schema - should be marked as so, but please validate and mark accordingly).

Reply:
Secrets show output does not contain any fields marked as sensitive in the resource files for secrets.
Password, secret_string and oauth_refresh_token are not outputed with SHOW